src/propIs.js

```diff
@@ -0,0 +1,28 @@
+var _curry3 = require('./internal/_curry3');
+var is = require('./is');
+var propSatisfies = require('./propSatisfies');
+
+
+/**
+ * Returns `true` if the specified object property is of the given type;
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category Type
+ * @sig Type -> String -> Object -> Boolean
+ * @param {Function} type
+ * @param {String} name
+ * @param {*} obj
+ * @return {Boolean}
+ * @see R.is
+ * @see R.propSatisfies
+ * @example
+ *
+ *      R.propIs('x', Number, {x: 1, y: 2});  //=> true
+ *      R.propIs('x', Number, {x: 'foo'});    //=> false
+ *      R.propIs('x', Number, {});            //=> false
+ */
+module.exports = _curry3(function propIs(type, name, obj) {
+  return propSatisfies(is(type), name, obj);
+});
```

Comment:
I just find this somewhat amusing, because the behaviour of `compose`/`pipe` seemed obvious when we wrote them, and it was fine until I tried to write `project`.  Then the missing features made me think a bit about how `compose`/`pipe` should work, but I couldn't think of any obvious way to fix them, and that was the introduction of `useWith`.  

This suggestion would mostly remove the need for `useWith`, which I think is great.

So the main thing I think about it when I consider this is how it takes the place of `useWith`.  But as flexible as `useWith` is, this suggestion is more flexible still.  I'm fascinated, but also noting how our recent trend has seemed to be sacrificing some flexibility in favor of simplicity, I'm simply a bit wary of extreme flexibility.